services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/logging/HankeKayttajaLoggingService.kt

```diff
@@ -0,0 +1,56 @@
+package fi.hel.haitaton.hanke.logging
+
+import fi.hel.haitaton.hanke.application.Application
+import fi.hel.haitaton.hanke.permissions.KayttajaTunniste
+import fi.hel.haitaton.hanke.permissions.PermissionEntity
+import fi.hel.haitaton.hanke.permissions.Role
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Propagation
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class HankeKayttajaLoggingService(private val auditLogService: AuditLogService) {
```

Comment:
Are all these functions unit tested?

Reply:
Oops, some of these were copied over and unchanged. Removing them.

I added unit tests for the ones that still in use.